### DISCUSSION TITLE
MIT Unified Momentum Model

### DISCUSSION THREAD

**Discussion by Edudiro** [NatLabRockies/floris#973] [Ideas]
Hello! 

Have you read the latest paper from MIT on "Unified momentum model for rotor aerodynamics across operating regimes"?

What is your opinion on it? Do you think, a priori, it could be implemented as an additional wake model and tested within the FLORIS framework?

https://www.nature.com/articles/s41467-024-50756-5

**Comment by misi9170** [comment]
Hi @Edudiro ,

Yes, the work Liew, Heck, and Howland are doing on this is really interesting! Getting this into FLORIS is already in the works---see #924. It will probably be a couple of months yet before we have worked through all of the finer details and have this included in the main FLORIS branch, but it is on its way!

Along the same lines, if you're interested in new models for how turbines operate in yaw-misaligned conditions, I'd also encourage you to check out the work of [Tamaro et al. (2023)](https://wes.copernicus.org/articles/9/1547/2024/wes-9-1547-2024-discussion.html). They present a similar theory to the work from Howland's group, and this is also going to be incorporated into FLORIS---see #832. Again, it will be a couple of months before this is in, and I'll likely be working with the authors of both pull requests at the same time to ensure that the necessary software is consistent across both models.

Note that these will primarily be "operation models", which describe the operation of the turbine (in particular, it's thrust, power, and axial induction factor), rather than "wake models". However, through the thrust and axial induction factor, they will also affect wake properties.

Misha

**Reply by Edudiro** [reply]
Thanks for your reponse. This is exciting!
